src/EditorFeatures/Core/FindUsages/IFindUsagesContext.cs

```diff
@@ -23,6 +23,7 @@ internal interface IFindUsagesContext
 
         Task OnDefinitionFoundAsync(DefinitionItem definition);
         Task OnReferenceFoundAsync(SourceReferenceItem reference);
+        Task OnExternalReferenceFoundAsync(ExternalReferenceItem reference);
```

Comment:
i needed to add a new entyrpoint to our FindRefs window to say "show this external item".  The core problem here is that the existing SourceReferenceItem is very tied to the roslyn workspace/solution model.  i.e. it references Documents directly and so on.  ExternalRefereneItem is the very looseygoosey reference that we'll just show, but entirely keep our hands off otherwise.

Reply:
I admit I'm not entirely convinced a base class isn't right here...